### PR TITLE
Reorganize menus

### LINKED ISSUE #64: Create User menu, move existing entries and add new ones
**Is your feature request related to a problem? Please describe.**
As a logged in user, I get logout and profile links. If I'm a member of the committee, I also get the session selection entry. If I'm a staff member, I get the checkin link. Additionally, speakers should have prominent links to manage their submissions.

**Describe the solution you'd like**
Create a User menu (preferably with a well-recognized icon) and put all these functions under it. Since we have more space in that menu, we can use more descriptive menu titles.

### PR DESCRIPTION
**Pull request by stefanbethke**
Functions available only to staff are now in the admin menu.

Functions available only to certain user classes (speakers, committee) 
are now in the user profile menu.

The user profile menu is represented by an icon; this uses Font Awesome.

Also corrected some translations.

This closes #64.

### CODE REVIEW

**Comment by stefanbethke** [comment]
Also closes #82, #61

File: devday/devday/cms_menus.py [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+        if post_cut:
+            event = Event.objects.current_event()
+            self.prune_all_nodes(nodes, request, event)
+        return nodes
+
+
+# It appears Django CMS has no easy way to add menu items in an ordered
+# way when using multiple register_menu classes.  To get the desired order,
+# this class contains all the menus across all applications.
+
```
Comment (jandd) [inline devday/devday/cms_menus.py]: As far as I understood the code it is ordered by the order in the INSTALLED_APPS setting. I like this more explicit approach though.

File: devday/devday/cms_toolbars.py [hunk 2]
```diff
@@ -1,18 +1,24 @@
...
 from cms.toolbar_pool import toolbar_pool
 from django.core.urlresolvers import reverse
 from django.utils.translation import ugettext_lazy as _
 
+from devday.utils.devday_toolbar import DevDayToolbar
+
 
 @toolbar_pool.register
-class DevDayToolbar(CMSToolbar):
+class DevDayToolbar(DevDayToolbar):
```
Comment (jandd) [inline devday/devday/cms_toolbars.py]: I think using the same class name for the child class here is not a good idea.

File: devday/event/locale/de/LC_MESSAGES/django.po [hunk 3]
```diff
@@ -7,82 +7,80 @@ msgid ""
...
-#: event/models.py:68
+#: event/models.py:67
 msgid "Event"
 msgstr "Veranstaltung"
 
-#: event/models.py:76
+#: event/models.py:75
 msgid "Registration Count"
 msgstr "Anmeldungen"
+
```
Comment (jandd) [inline devday/event/locale/de/LC_MESSAGES/django.po]: obsolete entries can be removed by `./run.sh manage makemessages -l de --no-obsolete`

File: devday/devday/cms_toolbars.py (hunk 2 above)
Comment (stefanbethke) [reply devday/devday/cms_toolbars.py]: That is confusing, true. I've renamed the base class DevDayToolbarBase.

File: devday/devday/cms_menus.py (hunk 1 above)
Comment (stefanbethke) [reply devday/devday/cms_menus.py]: I've amended the comment.

File: devday/event/locale/de/LC_MESSAGES/django.po (hunk 3 above)
Comment (stefanbethke) [reply devday/event/locale/de/LC_MESSAGES/django.po]: I've added ./run.sh messages with those parameters.

**Pull request merged**